packages/utility-features/kube-api/src/kube-api-parse.test.ts

```diff
@@ -103,129 +131,16 @@ const tests: KubeApiParseTestData[] = [
-      apiPrefix: "/apis",
-      apiGroup: "cluster.k8s.io",
-      apiVersion: "v1alpha1",
-      apiVersionWithGroup: "cluster.k8s.io/v1alpha1",
-      resource: "clusters",
-      name: undefined,
-      namespace: undefined,
-    },
-  ],
-
-  [
-    "/api/v1/namespaces/kube-system/pods/coredns-6955765f44-v8p27",
-    {
-      apiBase: "/api/v1/pods",
-      apiPrefix: "/api",
-      apiGroup: "",
-      apiVersion: "v1",
-      apiVersionWithGroup: "v1",
-      namespace: "kube-system",
-      resource: "pods",
-      name: "coredns-6955765f44-v8p27",
-    },
-  ],
-
-  [
-    "/apis/apps/v1/namespaces/default/deployments/some-deployment",
-    {
-      apiBase: "/apis/apps/v1/deployments",
-      apiGroup: "apps",
-      apiPrefix: "/apis",
-      apiVersion: "v1",
-      apiVersionWithGroup: "apps/v1",
-      name: "some-deployment",
-      namespace: "default",
-      resource: "deployments",
-    },
-  ],
-
-  [
-    "/apis/apiextensions.k8s.io/v1beta1/customresourcedefinitions/prometheuses.monitoring.coreos.com",
-    {
-      apiBase: "/apis/apiextensions.k8s.io/v1beta1/customresourcedefinitions",
-      apiPrefix: "/apis",
-      apiGroup: "apiextensions.k8s.io",
-      apiVersion: "v1beta1",
-      apiVersionWithGroup: "apiextensions.k8s.io/v1beta1",
-      namespace: undefined,
-      resource: "customresourcedefinitions",
-      name: "prometheuses.monitoring.coreos.com",
-    },
-  ],
-
-  [
-    "/api/v1/namespaces/kube-system/pods",
-    {
-      apiBase: "/api/v1/pods",
-      apiPrefix: "/api",
-      apiGroup: "",
-      apiVersion: "v1",
-      apiVersionWithGroup: "v1",
-      namespace: "kube-system",
-      resource: "pods",
-      name: undefined,
-    },
-  ],
-
-  [
-    "/apis/cluster.k8s.io/v1/namespaces/kube-system/pods",
-    {
-      apiBase: "/apis/cluster.k8s.io/v1/pods",
-      apiPrefix: "/apis",
-      apiGroup: "cluster.k8s.io",
-      apiVersion: "v1",
-      apiVersionWithGroup: "cluster.k8s.io/v1",
-      namespace: "kube-system",
-      resource: "pods",
-      name: undefined,
-    },
-  ],
 ];
 
-const invalidTests = [
-  undefined,
-  "",
-  "some-invalid-path",
-  "//apiextensions.k8s.io/v1beta1/customresourcedefinitions/prometheuses.monitoring.coreos.com",
-  "/apis//v1beta1/customresourcedefinitions/prometheuses.monitoring.coreos.com",
-  "/apis/apiextensions.k8s.io//customresourcedefinitions/prometheuses.monitoring.coreos.com",
-  "/apis/apiextensions.k8s.io/v1beta1//prometheuses.monitoring.coreos.com",
-  "/apis/apiextensions.k8s.io/v1beta1/customresourcedefinitions/",
-
-  "//v1beta1/customresourcedefinitions/prometheuses.monitoring.coreos.com",
-  "/api//v1beta1/customresourcedefinitions/prometheuses.monitoring.coreos.com",
-  "/api//customresourcedefinitions/prometheuses.monitoring.coreos.com",
-  "/api/v1beta1//prometheuses.monitoring.coreos.com",
-  "/api/v1beta1/customresourcedefinitions/",
-];
+const invalidTests = [undefined, "", "ajklsmh"];
```

Comment:
Here `tests` and `invalidTests` have degenerated to fewer scenarios. The commit name doesn't hint the motivation behind this. The missing scenarios describe an actual bug that was in production.